proxy/grpcproxy/watch_broadcast.go

```diff
@@ -54,13 +54,18 @@ func newWatchBroadcast(wp *watchProxy, w *watcher, update func(*watchBroadcast))
 		defer close(wb.donec)
 		// loop because leader loss will close channel
 		for cctx.Err() == nil {
-			wch := wp.cw.Watch(cctx, w.wr.key,
+			opts := []clientv3.OpOption{
 				clientv3.WithRange(w.wr.end),
 				clientv3.WithProgressNotify(),
-				clientv3.WithCreatedNotify(),
 				clientv3.WithRev(wb.nextrev),
 				clientv3.WithPrevKV(),
-			)
+			}
+			if wb.responses == 0 {
+				// don't send new creates from leader loss
```

Comment:
i think this is a more generic case? as long as this broadcast has sent out response, we should not add createNotify? Right now it is just for recreating the watcher for leader loss?